torch/csrc/jit/serialization/export_module.cpp

```diff
@@ -277,6 +278,8 @@ class ScriptModuleSerializer {
 
   void writeByteCode(const Module& module) {
     std::vector<c10::IValue> elements;
+    elements.emplace_back(
+        static_cast<int64_t>(caffe2::serialize::kProducedBytecodeVersion));
```

Comment:
We're writing a version number for every module? We want a single version for the whole saved model, right? 

Couple followups:
1. Where does `kProducedFileFormatVersion` get saved?
2. When we save bytecode, do both `kProducedFileFormatVersion` and `kProducedBytecodeVersion` get saved, or just the latter?

Reply:
Yes we have single version for the whole saved model. writeByteCode is called inside serialize(), which is called only once for the top-level module. 
1. It's saved in at the beginning of the bytecode section. Please refer to the summary of this PR for the example.
2. Only the latter is saved.

Reply:
Got it, thanks!